### DISCUSSION TITLE
Removing Dependencies

### DISCUSSION THREAD

**Discussion by nwns** [renovatebot/renovate#17955] [Suggest an Idea]
Is there a way to configure renovate to propose pr's that remove dependencies? The documentation for `replacementName` and `replacementVersion` don't indicate eitherway.

As a concrete story: We self host renovate for Gitlab.  My org uses primarily python with poetry as the package manager and VS Code as the IDE.  A few months ago VS Code added native support for the black code formatter.  This means that we do not need black as a development dependency any longer but we now have black listed as a dev dependency in many repos.  Currently when renovate creates a pr to update black we normally remove it, but it would be better if renovate could create prs that just remove dependencies that we have configured in our template to be removed instead.

**Comment by viceice** [comment]
not supported
